sllurp/llrp.py

```diff
@@ -853,6 +861,12 @@ def send_SET_READER_CONFIG(self, onCompletion):
                 },
             }
         }
+        keepalive_spec = {
+         'KeepaliveTriggerType': 1,
+         'TimeInterval': llrp_keepalive_interval_ms
+        }
+        if(llrp_keepalive_interval_ms != 0):
+            msg['SET_READER_CONFIG']['KeepaliveSpec'] = keepalive_spec
```

Comment:
It is probably better to use a "> 0" in the if, as we don't really want to handle any negative value.